blueocean-rest/README.md

```diff
@@ -58,6 +58,9 @@
     - [Get steps for a Pipeline node](#get-steps-for-a-pipeline-node)
     - [Get a Pipeline step details](#get-a-pipeline-step-details)
     - [Get Pipeline Steps](#get-pipeline-steps)
+    - [Get Pipeline Steps with Input](#get-pipeline-steps-with-input)
```

Comment:
Can this be updated to master - I was getting errors due to the missing github and git pipeline plugins with my workspace. Want to make sure it is ok. It seems to work fine and ATH passes, but if can be updated to master, then it can go in